validator/tests/test_scheduler/tests.py

```diff
@@ -43,7 +43,56 @@
 
 class TestSchedulersWithYaml(unittest.TestCase):
 
-    def test_single_block_files_individually(self):
+    def _setup_serial_scheduler(self):
+        context_manager = self._context_manager
+        squash_handler = context_manager.get_squash_handler()
+        first_state_root = context_manager.get_first_root()
+        scheduler = SerialScheduler(squash_handler,
+                                    first_state_root,
+                                    always_persist=False)
+        return context_manager, scheduler
+
+    def _setup_parallel_scheduler(self):
+        context_manager = self._context_manager
+        squash_handler = context_manager.get_squash_handler()
+        first_state_root = context_manager.get_first_root()
+        scheduler = ParallelScheduler(squash_handler,
+                                      first_state_root,
+                                      always_persist=False)
+        return context_manager, scheduler
+
+    def test_parallel_simple_scheduler_test(self):
```

Comment:
@peterschwarz I am going to put the parallel and serial tests under one test method, and then I'll put the docstring on the test method.